### ISSUE TITLE
InventoryControllerTest: test setup code

### ISSUE THREAD

**Issue by drewburlingame** [dedreira/InventoryExercise#1]
all tests appear to have the same setup code. what was the reasoning to copy/paste instead of moving the duplicate code into a single method?

**Comment by dedreira** [comment]
Yes, you’re right; the best option would have been to move duplicated code into a single method. At the time of building the tests I didn’t realize it.
